src/Microsoft.OpenApi/Writers/IOpenApiWriter.cs

```diff
@@ -39,7 +39,7 @@ public interface IOpenApiWriter
         /// <summary>
         /// Write the string value.
         /// </summary>
-        void WriteValue(string value);
+        void WriteValue(string? value);
```

Comment:
here I still don't understand why only this one would be nullable and not others? Should we revert that and have the caller type check?